kotlinpoet/src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -378,26 +378,52 @@ class FunSpec private constructor(
       parameters += parameterSpec
     }
 
+    @JvmName("callThisConstructorWithStrings")
```

Comment:
I really don't like the name and I can't see a better alternative. I think we should just drop the overloads that require changing method names.